app/src/main/java/com/mk/tasky/authentication/registration/presentation/RegistrationEvent.kt

```diff
@@ -0,0 +1,9 @@
+package com.mk.tasky.authentication.registration.presentation
+
+sealed class RegistrationEvent {
+    data class OnNameChange(val name: String) : RegistrationEvent()
+    data class OnEmailChange(val email: String) : RegistrationEvent()
+    data class OnPasswordChange(val password: String) : RegistrationEvent()
+    object ChangePasswordVisibility : RegistrationEvent()
```

Comment:
Change to what? :D Suggestion: `TogglePasswordVisibility` if it doesn't take arguments 😄